common/cb_types/gw_cb_types.txt

```diff
@@ -0,0 +1,63 @@
+# CB for gnoll war
+gnoll_war = {
```

Comment:
Ooof, it's buggy. Answering your comment...

> Make brightwood pack and riverpaw pack tributaries of redridge pack AND gain/lose land upon gnoll war CB (i dont know how to do this at at all so i need help.

I recommend using `tribal_invasion` in the war and deleting this CB. It's not EU4 so there's no good way to give lands to your allies.